etc/org.opencastproject.videoeditor.impl.VideoEditorServiceImpl.cfg

```diff
@@ -13,6 +13,12 @@ video.fade = 0.2
 # Default: 2000 milliseconds
 #segments.min.cut.duration = 2000
 
+# The main flavor of the vtt files which should be cut in "Shorten" mode.
+# The default mode will remove cues from vtt files if they overlap too much with a cut segment. This is not desirable
+# for e.g. chapters. "Shorten" mode will instead keep the cue, but reduce its duration by the overlap amount. A cue that
+# overlaps a cut segment completely is still removed.
+vtt.shorten.flavor.types = chapters
```

Comment:
Reminder from Dec 16: It should be clarified that this can be a list, and what the options (and formatting!) for that list are.